snow/consensus/snowball/nnary_snowflake.go

```diff
@@ -67,6 +67,15 @@ func (sf *nnarySnowflake) RecordSuccessfulPoll(choice ids.ID) {
 	sf.nnarySlush.RecordSuccessfulPoll(choice)
 }
 
+func (sf *nnarySnowflake) RecordPollPreference(choice ids.ID) {
+	if sf.finalized {
+		return // This instance is already decided.
+	}
+
+	sf.confidence = 0
```

Comment:
nit: again maybe it's trivial but would love a comment about why a preference poll wipes confidence. I guess people confidence increases if they receive a preference. People are definitely not nnarySnowflake instances

Reply:
I feel like this is documented by the method description (on the interface)